text_recognizer/stems/paragraph.py

```diff
@@ -18,28 +18,51 @@ class ParagraphStem(ImageStem):
 
     scale_factor = IMAGE_SCALE_FACTOR
 
-    def __init__(self, augment=False, color_jitter_kwargs=None, random_affine_kwargs=None):
+    def __init__(
+        self,
+        augment=False,
+        color_jitter_kwargs=None,
+        random_affine_kwargs=None,
+        random_perspective_kwargs=None,
+        gaussian_blur_kwargs=None,
+        sharpness_kwargs=None,
+    ):
         super().__init__()
 
         if not augment:
             self.pil_transforms = transforms.Compose([transforms.CenterCrop(IMAGE_SHAPE)])
         else:
             if color_jitter_kwargs is None:
-                color_jitter_kwargs = {"brightness": (0.8, 1.6)}
+                color_jitter_kwargs = {"brightness": 0.4, "contrast": 0.4}
```

Comment:
I feel like the jittering of the brightness and contrast could be turned up.

Reviewing examples from [the logged training predictions](https://wandb.ai/fullstackdeeplearning/fsdl-text-recognizer-2022-training/artifacts/run_table/run-2trhtbd4-trainpredictions/v98/files/train/predictions.table.json), it seems like the luminance levels are pretty uniform, and I suspect this is a big reason for our failure to generalize outside of the test distribution.

Reply:
I would suggest playing around with these values in a tight loop in a notebook until the outputs look nicely diverse. DM me to schedule a pairing session if you want!

Reply:
But note that images logged on W&B are generated from _float tensors_, which means that there's an auto-rescale on the to get uint8 from 0-255.

Reply:
(FROM SYNC MEETING) Current jitter is reducing the brightness of text more than it is increasing the brightness of the bg.

Reply:
didn't do much here